tests/CMakeLists.txt

```diff
@@ -250,6 +254,9 @@ add_test(NAME testLBFGS
          ${CMAKE_CURRENT_SOURCE_DIR}/LBFGS/coords.in
          ${CMAKE_CURRENT_SOURCE_DIR}/LBFGS/lrs.in
          ${CMAKE_CURRENT_SOURCE_DIR}/../potentials)
+add_test(NAME testVariableSizeMatrix
```

Comment:
Since it is a unit test (based on running a new executable), I would place it with the other unit tests, that is before the regression tests (based on running MGmol).

Reply:
Good idea. I made the change. Thanks